tests/layer_tests/pytorch_tests/test_pad.py

```diff
@@ -244,3 +244,44 @@ def test_reflection_padnd(self, pads, dtype, ie_device, precision, ir_version):
         print(ndim)
         self._test(*self.create_model(pads), ie_device, precision, ir_version,
                    kwargs_to_prepare_input={"ndim": ndim, "dtype": dtype})
+
+class TestReplicatePad(PytorchLayerTest):
```

Comment:
let us have separate test suite for each 1d, 2d, and 3d cases.

Reply:
done